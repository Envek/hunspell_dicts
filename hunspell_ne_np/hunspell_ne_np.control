# hunspell_ne_np extension
comment = 'Nepali Hunspell Dictionary'
default_version = '1.0'
module_pathname = '$libdir/hunspell_ne_np'
relocatable = true