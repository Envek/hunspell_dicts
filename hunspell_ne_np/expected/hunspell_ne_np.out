CREATE EXTENSION hunspell_ne_np;
CREATE TABLE table1(name varchar);
INSERT INTO table1 VALUES ('अँगअँकाउछन्'),
	('PostgreSQL संसारको सबैभन्दा उन्नत खुला स्रोत डाटास हो');
SELECT d.* FROM table1 AS t, LATERAL ts_debug('nepali_hunspell', t.name) AS d;
   alias   |    description    |   token    |   dictionaries    |   dictionary    | lexemes 
-----------+-------------------+------------+-------------------+-----------------+---------
 word      | Word, all letters | अँगअँकाउछन्   | {nepali_hunspell} | nepali_hunspell | 
 asciiword | Word, all ASCII   | PostgreSQL | {nepali_hunspell} | nepali_hunspell | 
 blank     | Space symbols     |            | {}                |                 | 
 word      | Word, all letters | संसारको     | {nepali_hunspell} | nepali_hunspell | {संसार}
 blank     | Space symbols     |            | {}                |                 | 
 word      | Word, all letters | सबैभन्दा     | {nepali_hunspell} | nepali_hunspell | {सबै}
 blank     | Space symbols     |            | {}                |                 | 
 word      | Word, all letters | उन्नत       | {nepali_hunspell} | nepali_hunspell | {उन्नत}
 blank     | Space symbols     |            | {}                |                 | 
 word      | Word, all letters | खुला        | {nepali_hunspell} | nepali_hunspell | {खुला}
 blank     | Space symbols     |            | {}                |                 | 
 word      | Word, all letters | स्रोत       | {nepali_hunspell} | nepali_hunspell | {स्रोत}
 blank     | Space symbols     |            | {}                |                 | 
 word      | Word, all letters | डाटास      | {nepali_hunspell} | nepali_hunspell | 
 blank     | Space symbols     |            | {}                |                 | 
 word      | Word, all letters | हो         | {nepali_hunspell} | nepali_hunspell | {हो}
(16 rows)

CREATE INDEX name_idx ON table1 USING GIN (to_tsvector('nepali_hunspell', "name"));
SELECT * FROM table1 WHERE to_tsvector('nepali_hunspell', name)
	@@ to_tsquery('nepali_hunspell', 'अँगअँकाउछन्');
NOTICE:  text-search query contains only stop words or doesn't contain lexemes, ignored
 name 
------
(0 rows)

SELECT * FROM table1 WHERE to_tsvector('nepali_hunspell', name)
	@@ to_tsquery('nepali_hunspell', 'स्रोत');
                      name                       
-------------------------------------------------
 PostgreSQL संसारको सबैभन्दा उन्नत खुला स्रोत डाटास हो
(1 row)

DROP INDEX name_idx;
CREATE INDEX name_idx ON table1 USING GIST (to_tsvector('nepali_hunspell', "name"));
SELECT * FROM table1 WHERE to_tsvector('nepali_hunspell', name)
	@@ to_tsquery('nepali_hunspell', 'संसारको');
                      name                       
-------------------------------------------------
 PostgreSQL संसारको सबैभन्दा उन्नत खुला स्रोत डाटास हो
(1 row)

SELECT * FROM table1 WHERE to_tsvector('nepali_hunspell', name)
	@@ to_tsquery('nepali_hunspell', 'हो');
                      name                       
-------------------------------------------------
 PostgreSQL संसारको सबैभन्दा उन्नत खुला स्रोत डाटास हो
(1 row)

